### DISCUSSION TITLE
Refunds

### DISCUSSION THREAD

**Discussion by TechyChap** [abuzuhri/Amazon-SP-API-CSharp#183] [General]
Not even sure if this is possible with the API but in seller central it is possible to initiate a refund. Anyone know which API does that?

**Comment by J-W-Chan** [comment]
you can try this feedType: [POST_PAYMENT_ADJUSTMENT_DATA](https://images-na.ssl-images-amazon.com/images/G/01/rainier/help/xsd/release_4_1/OrderAdjustment.xsd)
